examples/docs-examples/examples/vpn-network/inbound_proxy/read.me

```diff
@@ -0,0 +1,7 @@
+run requestor with:
```

Comment:
Rename the file to `README.md` and then takec are about nice formatting inside.

Reply:
fixed